### ISSUE TITLE
utils.js wrapText

### ISSUE THREAD

**Issue by aloulouamine** [openlayers/ol-mapbox-style#249]
Hello guys,

![image](https://user-images.githubusercontent.com/7517433/73671560-784ebf00-46ab-11ea-9349-fd361f5b9400.png)

A short text with space that doesn't exceed the line max wdith concatenate undefined to the current line.

Issue is caused by this line.
[https://github.com/openlayers/ol-mapbox-style/blob/master/util.js#L96](url)

**Comment by ahocevar** [comment]
Thanks for reporting this @aloulouamine. Can you please provide the exact text that caused this behavior? Then we can create a test case and fix the issue.

**Comment by aloulouamine** [comment]
@ahocevar yes you can take "91 A" as example.
Thank you

**Comment by ahocevar** [comment]
I cannot reproduce it. Can you please provide a working live example?

**Comment by mikeydunnx** [comment]
For what it's worth I am running into the same problem. My arguments are:
```
0: "Lazy H"
1: "normal 600 12px/1.2 "Open Sans""
2: 10
3: 0
```

I've pulled down the `better-wrapping` branch and it seems to have solved the problem. Any ETA on a release for this?

**Comment by ahocevar** [comment]
Are you sure? That branch has been merged a long time ago already.

**Comment by mikeydunnx** [comment]
Yes, Could be the issue was introduced after that branch was merged. I haven't tried stepping back through all the versions. I'll let you know if I can find when the bug was created.

**Comment by ahocevar** [comment]
@mikeydunnx Thanks for the function arguments, that makes it easier to write a test.

**Comment by mikeydunnx** [comment]
Here is the commit: f324384

No problem. This library is very helpful to us and your support is much appreciated.

**Comment by mikeydunnx** [comment]
opened #256

**Comment by ahocevar** [comment]
Fixed with #256
